src/styles/card.scss

```diff
@@ -0,0 +1,110 @@
+.card {
+  width: 200px;
+  padding: 15px 14px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-family: "Roboto", sans-serif;
```

Comment:
I would recommend storage all informations about fonts in separate file like typography.scss 
https://medium.com/codyhouse/create-your-design-system-part-1-typography-7c630d9092bd